src/AngularPublic.js

```diff
@@ -265,5 +265,6 @@ function publishExternalAPI(angular) {
         $$cookieReader: $$CookieReaderProvider
       });
     }
-  ]);
+  ])
+  .info({ angularVersion: '"NG_VERSION_FULL"' });
```

Comment:
I tend to think it as the module version indeed. But it doesn't make any real difference. `angularVersion` works for me.